src/sass/_advantages.scss

```diff
@@ -0,0 +1,146 @@
+@use './variables' as var;
+@use './global';
+
+.advantages {
+  background-color: #fff5f6;
+
+  &__container {
+    @extend .container;
+    padding-top: 39px;
+    padding-left: 20px;
+
+    @media screen and (min-width: 768px) {
+      padding-top: 66px;
+      padding-left: 36px;
+    }
+
+    @media screen and (min-width: 1200px) {
+      padding-top: 66px;
+      padding-left: 115px;
+    }
+  }
+
+  &__heading {
+    display: none;
+  }
+
+  &__list {
+    list-style-type: none;
+    width: auto;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, 230px);
+    gap: 38px;
+
+    @media screen and (min-width: 768px) {
+      grid-template-columns: repeat(auto-fill, 180px);
+      gap: 12.8%;
+    }
+
+    @media screen and (min-width: 1200px) {
+      grid-template-columns: repeat(auto-fill, 330px);
+      gap: 2.5%;
+    }
+  }
+
+  &__box {
+    width: auto;
+  }
+
+  &__box:nth-child(1)::before,
+  &__box:nth-child(2)::before,
+  &__box:nth-child(3)::before {
+    display: inline-block;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: top;
+  }
+
+  &__box:nth-child(1)::before {
+    width: 33px;
+    height: 50px;
+    content: '';
+    background-image: url(/images/icon1.png);
+
+    @media (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx) {
+      background-image: url(/images/icon1@2x.png);
+    }
+  }
+
+  &__box:nth-child(2)::before {
+    width: 50px;
+    height: 50px;
+    content: '';
+    background-image: url(/images/icon2.png);
+
+    @media (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx) {
+      background-image: url(/images/icon2@2x.png);
+    }
+  }
+  &__box:nth-child(3)::before {
+    width: 44px;
+    height: 50px;
+    content: '';
+    background-image: url(/images/icon3.png);
+
+    @media (min-device-pixel-ratio: 2), (min-resolution: 192dpi), (min-resolution: 2dppx) {
+      background-image: url(/images/icon3@2x.png);
+    }
+  }
+
+  &__box-heading {
+    padding-top: 16px;
+    padding-bottom: 12px;
+    font-family: var.$brand-font-family;
+    color: #d41443;
+    text-transform: uppercase;
```

Comment:
`text-transform: uppercase;` jest tutaj zbędne. 'kg' w nagłówku powinno być małymi literami. Co prawda w Figmie w widoku mobilnym jest wielkimi ale to jest błąd, w widoku tabletowym i desktopowym już jest małymi.